kadi/commands/commands_v2.py

```diff
@@ -376,6 +376,16 @@ def update_archive_and_get_cmds_recent(
             # Cut observing commands
             bad = np.isin(cmds["scs"], [131, 132, 133])
             cmds = cmds[~bad]
+        elif load_name in not_run_loads["HRC"]:
+            # Cut HRC state-changing commands
+            date = not_run_loads["HRC"][load_name]
+            bad = (
+                (cmds["tlmsid"] == "224PCAON")
```

Comment:
OK so I'm a little confused by this.  Should the command be removed here or should Hrc24vOn_Transition be updated so it is a no-op if the 15v is off?  Or does it not matter?  Not trying to overcomplicate here but not sure what will make the most sense to us later.  Thanks!

Reply:
Updating `Hrc24vOn_Transition` like you said might work and might be slightly more faithful to what is really happening. That said, doing a state-dependent transition is complicated and will require spending several more hours on this code that might never even get used. So I'm going to say that perfect is the enemy of good here, I've already invested too much time on this.